src/test/java/im/tox/tox4j/ToxCoreTestBase.java

```diff
@@ -232,12 +232,8 @@ protected static void assumeIPv6() {
         assumeConnection("2001:4860:4860::8888", 53);
     }
 
-    @NotNull ToxCore bootstrap(boolean useIPv6, @NotNull ToxCore tox) throws ToxBootstrapException {
-        if (useIPv6) {
-            tox.bootstrap(node().ipv6, node().port, node().dhtId);
-        } else {
-            tox.bootstrap(node().ipv4, node().port, node().dhtId);
-        }
+    @NotNull ToxCore bootstrap(boolean useIPv6, boolean udpEnabled, @NotNull ToxCore tox) throws ToxBootstrapException {
+        tox.bootstrap(useIPv6 ? node().ipv6 : node().ipv4, udpEnabled ? node().udpPort : node().tcpPort, node().dhtId);
```

Comment:
Can you format this with one argument per line? This line and the one above now exceed 100 columns.